### PR TITLE
Overhaul bomber tutorial

### PR DESCRIPTION
**Pull request by AngelOnFira**
This will close GDT-82.

### CODE REVIEW

**Comment by AngelOnFira** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1** <a href="https://app.graphite.dev/github/pr/rivet-gg/examples/1?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #17** <a href="https://app.graphite.dev/github/pr/rivet-gg/examples/17?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #5** <a href="https://app.graphite.dev/github/pr/rivet-gg/examples/5?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rivet-gg/examples/1?utm_source=stack-comment).

**Comment by NathanFlurry** [comment]
@AngelOnFira is this intended as a draft or waiting for reviewers?

**Comment by AngelOnFira** [comment]
My bad, I need to be using draft/published PRs better. There are a few things left to get this ready for others I think.

File: godot/bomber/LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2024 Rivet
+Copyright (c) 2023 Rivet
```
Comment (AngelOnFira) [inline godot/bomber/LICENSE:3]:
```suggestion
Copyright (c) 2024 Rivet
```

**Pull request merged**